pyscripts/horton_create_identities.py

```diff
@@ -11,149 +11,96 @@
 import json
 import pathlib
 import traceback
+import argparse
 from colorama import init, Fore, Back, Style
 from os.path import dirname, join, abspath
 sys.path.insert(0, abspath(join(dirname(__file__), '../horton_helpers')))
 from service_helper import Helper
 
 class HortonCreateIdentities:
+    def __init__(self, args):
+        self.horton_create_identities(args)
 
-    def __init__(self, input_manifest_file, save_manifest_file=''):
-        init(convert=True)
-        self.horton_create_identities( input_manifest_file, save_manifest_file)
-
-    def horton_create_identities(self, input_manifest_file, save_manifest_file=''):
+    def horton_create_identities(self, save_manifest_file):
+        if sys.platform == 'win32':
+            init(autoreset=True, convert=True)
```

Comment:
Why do you only call this on win32?  convert=True is harmless on linux and authreset=true seems like something you would want on both platforms.